Documentation/Usage.md

```diff
@@ -37,27 +37,24 @@ that contains Swift sources and a `Package.swift` manifest file at its root.
 ### Creating a Library Package
 
 A library package contains code which other packages can use and depend on. To
-get started, create a directory and run `swift package init`:
+get started run `swift package init --name MyPackage`:
 
-    $ mkdir MyPackage
-    $ cd MyPackage
-    $ swift package init # or swift package init --type library
+    $ swift package init --name MyPackage # or swift package init --name MyPackage --type library
     $ swift build
```

Comment:
I think you'd need to `cd MyPackage` here no?

Reply:
Either that, or specify `--package-path` to the `swift build` invocation.

Reply:
Yeah. You do need to move into the package's directory after it's created, or use `--package-path`.

Reply:
right, I think the example should be updated accordingly